<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5U4E5L3" failures="1" tests="4" name="ErrorValidation Test" time="53.737" errors="0" timestamp="2024-07-07T01:31:16 IST">
  <testcase classname="RahulShettyAcademy.Tests.ErrorValidationTest" name="LogInErrorValidation" time="1.846"/>
  <testcase classname="RahulShettyAcademy.Tests.ErrorValidationTest" name="productErrorValidation" time="1.137">
    <skipped/>
  </testcase> <!-- productErrorValidation -->
  <testcase classname="RahulShettyAcademy.Tests.ErrorValidationTest" name="productErrorValidation" time="1.938">
    <skipped/>
  </testcase> <!-- productErrorValidation -->
  <testcase classname="RahulShettyAcademy.Tests.ErrorValidationTest" name="productErrorValidation" time="1.038">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at RahulShettyAcademy.pageobjects.productCataloguePage.addProductTocart(productCataloguePage.java:49)
at RahulShettyAcademy.Tests.ErrorValidationTest.productErrorValidation(ErrorValidationTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- ErrorValidation Test -->
